site/src/@ndlib/gatsby-theme-marble/components/Pages/IndexPage/index.js

```diff
@@ -2,16 +2,19 @@
 // eslint-disable-next-line no-unused-vars
 import React from 'react'
 import PropTypes from 'prop-types'
-import { jsx } from 'theme-ui'
+import { Styled, jsx } from 'theme-ui'
+import { Link } from 'gatsby'
 import HeroBox from 'components/Shared/HeroBox'
 import SearchBox from 'components/Shared/SearchBox'
 import CardGroup from 'components/Shared/CardGroup'
-import Card from 'components/Shared/Card'
+// import Card from 'components/Shared/Card'
```

Comment:
Still a work in progress, may re-add card here depending on some additional rewriting of the "browse" area.